### PR TITLE
added spk for monit

### PR DESCRIPTION
**Pull request by cytec**
like requested in https://github.com/SynoCommunity/spksrc/issues/1392

### CODE REVIEW

File: spk/monit/Makefile [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+.PHONY: monit_extra_install
+monit_extra_install:
+	install -m 755 -d $(STAGING_DIR)/app
+	install -m 644 src/app/config $(STAGING_DIR)/app/config
+	install -m 755 -d $(STAGING_DIR)/app/images
+	install -m 755 -d $(STAGING_DIR)/var
+	install -m 755 -d $(STAGING_DIR)/var/monit.d
+	install -m 755 -d $(STAGING_DIR)/var/events
+	install -m 700 src/monitrc $(STAGING_DIR)/var/monitrc
+	for size in 16 24 32 48; do \
```
Comment (Dr-Bean) [inline spk/monit/Makefile]: Missing `72`

**Comment by Dr-Bean** [comment]
Just scrolled past the code, and noticed a minor adjustment. Apart from that, with a squash we're good to go, I think :)

**Comment by cytec** [comment]
@Dr-Bean i'm going to change that later.

File: spk/monit/src/dsm-control.sh [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+PATH="${INSTALL_DIR}/bin:${PATH}"
+
+CFG_FILE="${INSTALL_DIR}/var/monitrc"
+PID_FILE="${INSTALL_DIR}/var/monit.pid"
+LOG_FILE="${INSTALL_DIR}/var/monit.log"
+
+start_daemon ()
+{
+    #run as root with verbose logging
+    PATH=${PATH} monit -c ${CFG_FILE} -vv -l ${LOG_FILE}
```
Comment (Diaoul) [inline spk/monit/src/dsm-control.sh]: Isn't verbose a problem with log file size?

Comment (cytec) [inline spk/monit/src/dsm-control.sh]: yup. still a leftover from debugging :/
thanks

**Comment by cytec** [comment]
@Diaoul fixed it ;)

**Comment by Diaoul** [comment]
@cytec: after publish go to the admin interface of SynoCommunity to activate your builds. /admin at the end of the URL after a successful login at /login.

**Comment by cytec** [comment]
@Diaoul done. thought you maybe want to check before activating or something. nevermind :)

**Comment by Diaoul** [comment]
nope, if there's a bug that's on you :)

**Pull request merged**